fullstack-cert/html-projects/lab-movie-review-page/user-stories.md

```diff
@@ -0,0 +1,9 @@
+1. Your should have a `section` element with the class `review`.
+2. Inside the section element, you should have an `h1` element for the movie title.
+3. Below the `h1` element, you should have an `img` element displaying the movie cover with an `alt` attribute describing the image.
+4. You should have a `p` element containing a brief movie description.
+5. You should have a `label` element for movie rating, linked to a `meter` element using the `for` and `id` attributes.
+6. The `meter` element should display the movie rating on a scale of 0 to 10 and have a corresponding `p` element displaying the rating as text (e.g., "9.2/10").
```

Comment:
I left a note about the meter element earlier.

If it was not taught, then these will need to be removed